config/environments/production.rb

```diff
@@ -48,14 +47,14 @@
   # Uncomment if you wish to allow Action Cable access from any origin.
   config.action_cable.disable_request_forgery_protection = true
 
+	# Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  config.assume_ssl = true
```

Comment:
I turned this on, but not sure what the effect is

Reply:
Not sure what the benefit is from this setting.